<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VodPlayInterface</class>
 <widget class="QMainWindow" name="VodPlayInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>761</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>VodPlayInterface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1071</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="id_monitor_interface">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="WndPreview" name="id_monitor_frame_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>501</width>
        <height>161</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="id_monitor_frame_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>501</width>
        <height>181</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="id_monitor_frame_6">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>350</y>
        <width>501</width>
        <height>161</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="id_monitor_frame_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>501</width>
        <height>161</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="id_monitor_frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>501</width>
        <height>181</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="id_monitor_frame_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>501</width>
        <height>161</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="id_monitor_frame_8">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>510</y>
        <width>501</width>
        <height>161</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="WndPreview" name="id_monitor_frame_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>501</width>
        <height>161</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="WndConfig" name="id_config_interface">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="id_config_label_ir">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>红外设备</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_config_label_vi">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>可见光设备</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_config_label_pan">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>490</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>云台设备</string>
      </property>
     </widget>
     <widget class="PicPreview" name="id_config_frame_ir">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>721</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="PicPreview" name="id_config_frame_vi">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>721</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="PicPreviewPan" name="id_config_frame_pan">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>510</y>
        <width>721</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="PicPreviewGroup" name="id_config_frame_group">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>671</height>
       </rect>
      </property>
      <widget class="QPushButton" name="id_config_save">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>290</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_config_change">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>400</y>
         <width>290</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="id_config_label_move">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>-10</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>移动显示效果窗口</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_config_btn_save">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>700</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="id_analyse_interface">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="WndIRDisplay" name="id_analyse_frame_ir">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>411</width>
        <height>281</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="WndVIDisplay" name="id_analyse_frame_vi">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>361</width>
        <height>281</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>350</y>
        <width>781</width>
        <height>381</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="id_analyse_group_pan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>240</width>
        <height>240</height>
       </rect>
      </property>
      <property name="title">
       <string>云台控制</string>
      </property>
      <widget class="QPushButton" name="id_analyse_btn_rightup">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>61</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>右上</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_right">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>64</y>
         <width>61</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>右</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_down">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>108</y>
         <width>61</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>下</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_analyse_lineedit_preset">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>152</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_recall">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>196</y>
         <width>90</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>招回</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_left">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>64</y>
         <width>61</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>左</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_up">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>61</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>上</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_rightdown">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>108</y>
         <width>61</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>右下</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_leftup">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>左上</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_leftdown">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>108</y>
         <width>61</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>左下</string>
       </property>
      </widget>
      <widget class="QPushButton" name="id_analyse_btn_set">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>196</y>
         <width>90</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
      <widget class="QLabel" name="id_analyse_label_preset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>90</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>预置点：</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="id_analyse_group_connect_status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>240</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>连接状态</string>
      </property>
      <widget class="QTableWidget" name="id_analyse_widget_connect_status">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>220</width>
         <height>401</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>信息内容</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="WndVideoPlayback" name="id_back_interface">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="WndIRDisplay" name="id_back_frame_ir">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>391</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="WndVIDisplay" name="id_back_frame_vi">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>391</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QSlider" name="id_back_slider_play">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>290</y>
        <width>561</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="id_back_btn_start">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>▶</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_back_btn_replay">
      <property name="geometry">
       <rect>
        <x>959</x>
        <y>290</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>330</y>
        <width>791</width>
        <height>341</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="id_back_group_select">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>240</width>
        <height>240</height>
       </rect>
      </property>
      <property name="title">
       <string>选择</string>
      </property>
      <widget class="QTimeEdit" name="id_back_time_end">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="id_back_date_start">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="id_back_label_endtime">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>结束时间：</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="id_back_date_end">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="id_back_label_device">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备点：</string>
       </property>
      </widget>
      <widget class="QLabel" name="id_back_label_starttime">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开始时间：</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="id_back_time_start">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>109</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="id_back_combobox_device">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="id_back_group_connect_status">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>240</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>连接状态</string>
      </property>
      <widget class="QTableWidget" name="id_back_widget_connect_status">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>220</width>
         <height>361</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>信息内容</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QPushButton" name="id_back_btn_stop">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ST</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_back_btn_first_frame">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>|&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_back_btn_pre_frame">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>290</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_back_btn_next_frame">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>290</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_back_btn_last_frame">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;|</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_back_btn_play_speed">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>290</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WndPreview</class>
   <extends>QFrame</extends>
   <header location="global">wndpreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PicPreview</class>
   <extends>QFrame</extends>
   <header location="global">picpreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PicPreviewPan</class>
   <extends>QFrame</extends>
   <header location="global">picpreviewpan.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PicPreviewGroup</class>
   <extends>QFrame</extends>
   <header>picpreviewgroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WndConfig</class>
   <extends>QWidget</extends>
   <header>wndconfig.h</header>
  </customwidget>
  <customwidget>
   <class>WndIRDisplay</class>
   <extends>QFrame</extends>
   <header location="global">wndirdisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WndVIDisplay</class>
   <extends>QFrame</extends>
   <header location="global">wndvidisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WndVideoPlayback</class>
   <extends>QWidget</extends>
   <header location="global">wndvideoplayback.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
